C/ChibiScheme/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+using BinaryBuilder
+
+name = "ChibiScheme"
+version = v"0.10"
+
+# Collection of sources required to build NLopt
+sources = [
+    GitSource("https://github.com/ashinn/chibi-scheme.git",
+              "05eb4ebd357e2bf2fe5aa7da13b0422ce20ddf7f"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/chibi-scheme
+
+apk add chibi-scheme
+
+
+make -j${nproc} \
+    PREFIX="${prefix}" \
+    CC="${CC_BUILD}" \
+    AR="${AR_BUILD}" \
+    LD="${LD_BUILD}" \
+    RANLIB="${RANLIB_BUILD}" \
+    CHIBI=`which chibi-scheme` \
+    CHIBI_FFI=`which chibi-ffi` \
+    CHIBI_DOC=`which chibi-doc` \
+    SNOW_CHIBI=`which snow-chibi` \
+    SO=".${dlext}" \
+    CLIBFLAGS="-fPIC" \
+    CLINKFLAGS="-shared" \
+    STATICFLAGS="-static -DSEXP_USE_DL=0" \
+    LIBCHIBI_FLAGS="-Wl,-soname,libchibi-scheme.${dlext}.0" \
```

Comment:
I'm not sure this is necessary, we automatically set the soname for ELF and MachO objects during the audit anyway (and this format of the soname is usually wrong for macOS and Windows)

Reply:
removed